tests/test_0750-avoid-empty-TBasket-issue.py

```diff
@@ -0,0 +1,17 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/uproot4/blob/main/LICENSE
+
+import json
+import sys
+
+import numpy
+import pytest
+import skhep_testdata
+
+import uproot
+
+
+def test():
+    with uproot.open(skhep_testdata.data_path("uproot-issue-750.root"))[
```

Comment:
```suggestion
def test():
    pytest.importorskip("awkward")
    with uproot.open(skhep_testdata.data_path("uproot-issue-750.root"))[
```

Reply:
In `main-v4`, Awkward isn't installed by default, and the CI skips it for windows. Perhaps we still want to run this on windows. If so, is there a way to read but not interpret?

Reply:
Both of these should be using `library="np"`!